utils/jwt.ts

```diff
@@ -9,19 +9,51 @@ export function genJwt(id: string): string {
   return jwt.sign({id}, process.env.TOKEN_SECRET, { expiresIn: '7d' });
 }
 
-export function validateJwt(req: Request, res: Response, next: NextFunction) {
-  const authHeader = req.headers.authorization;
-  const token = authHeader && authHeader.split(' ')[1];
+interface validateJwtOptions {
+  req: Request;
+  res: Response;
+  next: NextFunction;
+  token?: string;
+  key: string;
+  tokenName: string;
+}
 
+function validateJwtToken({req, res, next, token, key, tokenName}: validateJwtOptions) {
   if (token == null) return res.sendStatus(401).end();
 
   jwt.verify(token, process.env.TOKEN_SECRET as string, (err: VerifyErrors, decoded: any) => {
     if (err) {
       logger.error(err);
-      return res.sendStatus(403).end();
+      return res.status(403).json(tokenName + ' is expired').end();
```

Comment:
```suggestion
      return res.status(403).json({key: true}).end();
```

Reply:
what is the true suppose to indicate? I feel like {lobbyId: true} or {id: true} isn't too insightful to the frontend. I wanted to use tokenName because I think "user id" is more descriptive than "id".

Reply:
in which case we should switch the key to be more descriptive!